src/dos/dos.cpp

```diff
@@ -32,6 +32,8 @@
 
 DOS_Block dos;
 DOS_InfoBlock dos_infoblock;
+int enablelfn=-1;
```

Comment:
It will eventually be used by src/dos.cpp, shell/shell_cmds.cpp and misc/programs.cpp (and gui/sdlmain.cpp as in DOSBox-X). What is your suggestion then?